src/CMakeLists.txt

```diff
@@ -87,9 +87,11 @@ if(APPLE)
     target_link_libraries(86Box Freetype::Freetype)
 endif()
 
-find_package(OpenAL REQUIRED)
-include_directories(${OPENAL_INCLUDE_DIR})
-target_link_libraries(86Box ${OPENAL_LIBRARY})
+if(OPENAL)
+    find_package(OpenAL REQUIRED)
```

Comment:
Perhaps the `find_package` call could be not `REQUIRED` and just have `OPENAL` be forced off if it's not found?

Reply:
I'm really not all that good with cmake build scripts, so if you wanna, go right ahead, it'd be very welcome. (: